src/PointwiseFunctions/GeneralRelativity/DerivativeSpatialChristoffel.hpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace gr {
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the spatial derivative of the spatial christoffel symbols of
+ * the second kind from the inverse spatial metric, the spatial derivative of
+ * the inverse spatial metric, the first spatial derivative of the spatial
+ * metric, and the second spatial derivative of the spatial metric.
+ *
+ * \details Computes the derivative as:
+ * \f{align}
+ *     \partial_k \Gamma^m{}_{ij} &=
+ *       \frac{1}{2} (\partial_k \gamma^{ij} (\partial_i \gamma_{jl} +
+ *       \partial_j \gamma_{il} - \partial_l \gamma_{ij}) +
+ *       \gamma^{ml}(\partial_{(k} \partial_{i)jl} +
+ *       \partial_{(k} \partial_{j)il} - \partial_{(k} \partial_{l)ij}))
```

Comment:
This notation is weird, you should have `\gamma_{il}` in here. 

I'm also not confident that you have the factors of 1/2 right here, I'm pretty sure you're missing a factor of 1/2. You should test this function by having a Christoffel second kind from say a Kerr-Schild analytic solution, taking its numerical derivative, and computing the derivative using your newly added function and comparing them. Currently all your test is checking is that you have the same mistake in both the python and the C++ code :)

Reply:
Yes, so to report back: yesterday I made the mistake of taking eq. (16) and reverse engineering it back into what I believed was the non-conformal form of the derivative. After just looking at the normal equation for a non-conformal christoffel symbol, I agree this generic implementation was missing a 1/2. I confirmed this by testing it against a numerical derivative as well.

Reply:
Ah good! :)